src/hotspot/share/gc/g1/g1RemSet.cpp

```diff
@@ -586,9 +580,9 @@ class G1RemSetSamplingTask : public G1ServiceTask {
     SuspendibleThreadSetJoiner sts;
 
     // Reschedule if a GC happened too recently.
-    if (should_reschedule()) {
-      // Calculate the delay given the last GC and the interval.
-      schedule(reschedule_delay_ms());
+    auto delay_ms = reschedule_delay_ms();
```

Comment:
I would prefer to be explicit about `jlong` since sometimes time comes in the form of `doubles`.